code/modules/medical/blood_system.dm

```diff
@@ -386,6 +386,8 @@ this is already used where it needs to be used, you can probably ignore it.
 	var/mob/living/H = some_idiot
 
 	var/blood_color_to_pass = DEFAULT_BLOOD_COLOR
+	var/reagents_to_transfer = (num_amount / 20)
```

Comment:
```suggestion
	var/reagents_to_transfer = (num_amount / 5) * 2
```

Reply:
This isn't a good thing, if you keep it as low as 2 units of chem per 3 of blood you are going to both reduce how much you bleed in general by 40%, overall nerfing everything that deals bleed, while also keeping what i can't stress enough is a completely unbalanced amount of chems lost every tick you are bleeding for what amounts to no drawback at all.

Reply:
The drawback is stabbing and doing damage to yourself while the poison continues to do damage and theres an antag who already tried to kill you watching you make yourself weaker

Reply:
The drawback is stabbing and doing damage to yourself while the poison continues to do damage and theres an antag who already tried to kill you watching you make yourself weaker

Reply:
> The drawback is stabbing and doing damage to yourself while the poison continues to do damage and theres an antag who already tried to kill you watching you make yourself weaker

again, if you want anything like that amount of chem flushing, calomel deals 5 tox per cycle, for it's 5 units of reduction per cycle, calomel is also a drug you have to get from medical or make yourself. A couple of stabs is 10 damage, and will keep removing units of all the chems ALL THE WHILE YOU ARE BLEEDING, you need to bleed about 200 units before things get dire, and that's A LOT of chems you can just get rid of with literally minimal consequences, this isn't close to being a fair thing at all, besides as i said, being a 40% nerf on literally everything that deals bleed.

Reply:
a bit late but this could also be rescaled to be the ratio of blood VS reagents in your bloodstream. fun stuff with low blood count like bleeding pure liquor.

Reply:
> a bit late but this could also be rescaled to be the ratio of blood VS reagents in your bloodstream. fun stuff with low blood count like bleeding pure liquor.

would be more interesting than just slashing the values by 80% at least